Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -14,11 +14,12 @@ Describe "Get-BqJob" {
         New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
             New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
             Add-BqTabledata $filename CSV -SkipLeadingRows 1
+        $table | Add-BqTabledata $filename CSV -SkipLeadingRows 1
     }
 
     It "should list jobs from the past 6 months" {
         $jobs = Get-BqJob
-        $jobs.Count | Should BeGreaterThan 0
+        $jobs.Count | Should BeGreaterThan 1
```

Comment:
This just tests that a job exists.  Is this something we need a test for?  We test the get calls below